src/components/LeftSidebar/ConversationsList/Conversation.vue

```diff
@@ -182,7 +182,7 @@ export default {
 
 	setup(props) {
 		const { item, isSearchResult } = toRefs(props)
-		const { counterType, conversationInformation } = useConversationInfo({ item, isSearchResult })
+		const { counterType, conversationInformation } = useConversationInfo({ item, isSearchResult, exposeMessages: true, exposeDescription: true })
```

Comment:
![image](https://github.com/nextcloud/spreed/assets/93392545/bd6a2786-65c6-4c99-bb2d-d7faa5f30aac)

Reply:
Maybe add 'No messages' as a fallback?

Reply:
It can be a bit confusing if we add a text there as hint for "No Messages", unless it has a different format